src/main/java/net/darksky/project/numberone/ReadAFile.java

```diff
@@ -0,0 +1,38 @@
+package net.darksky.project.numberone;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+
+public class ReadAFile {
```

Comment:
Where this class is used?

Reply:
this ReadAFile class is for to read a text that is in the beginning and end of the process. But when I create a .jar file, another user should include that text file in classpath so it will be discomfort for the user. Text will be printed using System.out.println() statement instead of file and this class's removed.